build.gradle

```diff
@@ -25,7 +25,6 @@ ext {
     minGroovyVersion = "2.5.2"
     groovyDependency = [
       "org.codehaus.groovy:groovy:${groovyVersion}",
```

Comment:
the `coreConsole` task depends on `groovysh`

Reply:
Good point. I've just reworked the approach to don't pollute the compile classpath.